contrib/blueflood-docker/docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 blueflood:
-  build: .
+  image: goru97/blueflood
```

Comment:
This has your name on it. Is it a path to github? Should we point to rackerlabs or a generic one?